docs/codes/Genji Days Of The Blade BCES00002 01.00.ncl

```diff
@@ -1,6 +1,14 @@
 Also known as
 Genji Kamiu Souran
 
+#
+Restore health
+0
+F2 10F8BAAC 10F8BAAE
```

Comment:
that `F2` code seems unsupported by Artemis... maybe it's for a different tool like netcheat ?

Reply:
It's from the documentation of the [netcheat file provided by you](https://github.com/bucanero/ArtemisPS3/blame/master/docs/INSTALLATION.md#L38) and also appears in [some other cheat codes](https://github.com/bucanero/ArtemisPS3/blob/master/docs/codes/Shin%20Sangoku%20Muou%20MultiRaid%20Special%20BLAS50163%2001.01.ncl#L28).

Reply:
Yes, my bad, I haven't been reviewing Artemis in a long time.

Reply:
These additional code types (like `F`) should be added to the Wiki:
https://github.com/bucanero/ArtemisPS3/wiki/Artemis-NetCheat-code-types

They are currently missing from the documentation.